From a714d5f3c35f8ef8442615ed551c5bc2edb05cba Mon Sep 17 00:00:00 2001
From: mhaquila <mhaquila@pharosmaris.net>
Date: Fri, 21 Jan 2022 13:05:24 +0100
Subject: [PATCH] fix: i18n.merge_file does not take positional arguments
Patch-Source: https://gitlab.gnome.org/World/apostrophe/-/merge_requests/241

Correct issues of data/meson.build: ERROR: Function does not take positional arguments.
---
 data/meson.build | 2 --
 1 file changed, 2 deletions(-)

diff --git a/data/meson.build b/data/meson.build
index ee19288..c25cbc6 100644
--- a/data/meson.build
+++ b/data/meson.build
@@ -2,7 +2,6 @@
 desktop_conf = configuration_data()
 desktop_conf.set('icon', application_id)
 desktop_file = i18n.merge_file(
-  'desktop',
   input: configure_file(
     input: 'org.gnome.gitlab.somas.Apostrophe.desktop.in.in',
     output: '@BASENAME@',
@@ -46,7 +45,6 @@ metainfo_configured = configure_file(
 # )
 
 appdata_file = i18n.merge_file(
-  'appdata',
   input: metainfo_configured,
   output: '@0@.metainfo.xml'.format(application_id),
   po_dir: podir,
-- 
